pom.xml

```diff
@@ -1080,14 +1080,14 @@
       </modules>
     </profile>
     <!--
-     Name:  wildfly27dev
-     Descr: Test with WFLY27 upstream version -->
+     Name:  wildfly28dev
+     Descr: Test with WFLY28 upstream version -->
     <profile>
-      <id>wildfly27dev</id>
+      <id>wildfly28dev</id>
       <properties>
         <jbossws.integration.target>wildfly2700</jbossws.integration.target>
```

Comment:
what about this value?

Reply:
probably refers to https://github.com/jimma/jbossws-cxf/tree/upgrade-wfly28/modules/resources/src/main/resources/modules

Reply:
Yes. That's it. We can probably remove this in the future： https://issues.redhat.com/browse/JBWS-4336.

Reply:
ok let's ignore that for now